app/models/privacy_policy.rb

```diff
@@ -16,7 +16,7 @@ def self.current
 
   def acceptance_required?(user)
     return false if !user || user.admin?
-    return false unless user.induction_coordinator_profile
+    return false unless user.induction_coordinator_profile || user.participant_profiles.active.ecf.any?
```

Comment:
I smell conflicts https://github.com/DFE-Digital/early-careers-framework/pull/795/files#diff-24c3894ada4eeac7641898881af8e0bad409577543af94ff786544e6ec5ac3a3R19

Reply:
Is the presence of a `teacher_profile` all we need here ?

Reply:
> Looks good to me, great work.
> 
> As final touches, does removing `rutabaga: false` break anything? If not, we could drop it from the tests you are adding :)
> 
> As for validations, I reckon at some point we might want to extract things like date or trn shape validation - kind of like what we do with email validations (to conform to notify standards). But we can wait with it until needed, and having all validations for a step in a single method is neat.

Yep, no need for `rutabaga: false` anymore :)